### PR TITLE
Chore add missing tests for force subtask results use case

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/107.

### CODE REVIEW

File: concent_api/core/tests/test_auth_accept_or_reject_results.py [hunk 1]
```diff
@@ -0,0 +1,1110 @@
...
+            requestor_public_key = self._get_encoded_requestor_public_key(),
+        )
+
+        # STEP 2: Different provider forces subtask results via Concent with message with the same task_id with different keys.
+        # Request is processed correctly.
+        different_serialized_force_subtask_results = self._get_serialized_force_subtask_results(
+            timestamp="2018-02-05 10:00:15",
+            ack_report_computed_task=self._get_deserialized_ack_report_computed_task(
+                timestamp="2018-02-05 10:00:15",
+                subtask_id="xxyyzz",
```
Comment (cameel) [inline concent_api/core/tests/test_auth_accept_or_reject_results.py]: Please fix your style here.

File: concent_api/core/tests/test_auth_accept_or_reject_results.py [hunk 2]
```diff
@@ -0,0 +1,1110 @@
...
+        )
+
+        with mock.patch('core.views.is_provider_account_status_positive', _get_provider_account_status_true_mock):
+            with freeze_time("2018-02-05 10:00:25"):
+                response = self.client.post(
+                    reverse('core:send'),
+                    data                                = serialized_force_subtask_results,
+                    content_type                        = 'application/octet-stream',
+                    HTTP_CONCENT_CLIENT_PUBLIC_KEY      = self._get_encoded_provider_public_key(),
+                    HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY=self._get_encoded_requestor_public_key(),
```
Comment (cameel) [inline concent_api/core/tests/test_auth_accept_or_reject_results.py]: Style.

File: concent_api/core/tests/test_auth_accept_or_reject_results.py [hunk 3]
```diff
@@ -0,0 +1,1110 @@
...
+
+        assert len(response.content) == 0
+        assert response.status_code  == 202
+
+        self._test_database_objects(
+            last_object_type         = message.concents.ForceSubtaskResults,
+            task_id                  = '2',
+            receive_delivered_status = False,
+        )
+        self._test_auth_message_counter_increased()
```
Comment (cameel) [inline concent_api/core/tests/test_auth_accept_or_reject_results.py]: These methods look like test assertions. Their names should start with `assert_` like they do for built-in assertions.

File: concent_api/core/tests/test_auth_accept_or_reject_results.py [hunk 4]
```diff
@@ -0,0 +1,1110 @@
...
+        )
+
+        with mock.patch('core.views.is_provider_account_status_positive', _get_provider_account_status_true_mock):
+            with freeze_time("2018-02-05 10:00:30"):
+                response = self.client.post(
+                    reverse('core:send'),
+                    data                                = serialized_force_subtask_results,
+                    content_type                        = 'application/octet-stream',
+                    HTTP_CONCENT_CLIENT_PUBLIC_KEY      = self._get_encoded_provider_public_key(),
+                    HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY=self._get_encoded_requestor_public_key(),
```
Comment (cameel) [inline concent_api/core/tests/test_auth_accept_or_reject_results.py]: Style.

File: concent_api/core/tests/test_auth_accept_or_reject_results.py [hunk 5]
```diff
@@ -0,0 +1,1110 @@
...
+                'timestamp':                            self._parse_iso_date_to_timestamp("2018-02-05 10:00:24"),
+                'ack_report_computed_task.timestamp':   self._parse_iso_date_to_timestamp("2018-02-05 10:00:15"),
+                'ack_report_computed_task.subtask_id':  'xxyyzz',
+            }
+        )
+        self._test_auth_message_counter_increased()
+        self._test_auth_message_last(
+            related_message=message.concents.ForceSubtaskResults,
+            provider_public_key=self._get_encoded_provider_public_key(),
+            requestor_public_key=self._get_encoded_requestor_public_key(),
```
Comment (cameel) [inline concent_api/core/tests/test_auth_accept_or_reject_results.py]: Style.

File: concent_api/core/tests/test_auth_accept_or_reject_results.py (hunk 1 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_auth_accept_or_reject_results.py]: :+1:

File: concent_api/core/tests/test_auth_accept_or_reject_results.py (hunk 2 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_auth_accept_or_reject_results.py]: :+1:

File: concent_api/core/tests/test_auth_accept_or_reject_results.py (hunk 4 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_auth_accept_or_reject_results.py]: :+1:

File: concent_api/core/tests/test_auth_accept_or_reject_results.py (hunk 5 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_auth_accept_or_reject_results.py]: :+1:

File: concent_api/core/tests/test_auth_accept_or_reject_results.py (hunk 3 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_auth_accept_or_reject_results.py]: :+1:

**Pull request merged**